pkg/markdown/markdown.go

```diff
@@ -92,6 +94,19 @@ func GetStyle(defaultStyle string) string {
 	return "notty"
 }
 
+func formatImgTags(content string) string {
+	lines := strings.Split(content, "\n")
```

Comment:
Lets avoid splitting the content into a slice of lines just to join it all together again. We can do regexp replacement on content without that.